awx/ui_next/src/screens/Inventory/shared/InventoryForm.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import { Formik, Field } from 'formik';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { func, number, shape } from 'prop-types';
+
+import { VariablesField } from '@components/CodeMirrorInput';
+import { Form } from '@patternfly/react-core';
+import FormField from '@components/FormField';
+import FormActionGroup from '@components/FormActionGroup/FormActionGroup';
+import FormRow from '@components/FormRow';
+import { required } from '@util/validators';
+import InstanceGroupsLookup from '@components/Lookup/InstanceGroupsLookup';
+import OrganizationLookup from '@components/Lookup/OrganizationLookup';
+import CredentialLookup from '@components/Lookup/CredentialLookup';
+
+function InventoryForm({
+  inventory = {},
+  i18n,
+  handleCancel,
+  handleSubmit,
```

Comment:
These prop names should probably be `onSubmit` and `onCancel` (though I realize we did it this way in some other forms)